packages/component/src/hooks/internal/useSettableDictateAbortable.js

```diff
@@ -0,0 +1,5 @@
+import { useState } from 'react';
+
+export default function useSettableDictateAbortable() {
+  return useState();
```

Comment:
Question, I don't understand what this is doing, could you explain?

Reply:
Stupid me. 😡

This should be a state stored in `Composer.js`, and this hook should be a way to retrieve that state.